### ISSUE TITLE
No images 

### ISSUE THREAD

**Issue by Willow-Rose** [fetlife/android#463]
A few days ago I had just a black screen. That was semi fixed because now I have usernames and wording, but no pictures are loading.

**Comment by Germanbight** [comment]
Same here

**Comment by johnbaku** [comment]
@Willow-Rose and @Germanbight if you update the app to the latest version does it fix the problem?

cc/ @kistitan

**Comment by Willow-Rose** [comment]
@johnbaku What is the latest version? I've installed, uninstalled, and reinstalled 2.15.5 about 5 times now.

**Comment by kistitan** [comment]
@Willow-Rose 
The latest stable version is indeed 2.15.5, but there was no changes in image displaying for very long.
So the issue should not be realted to that specific version.
We have a 2.16.0 preview version too, you can try to install that, but again no related code change for images. 

Can you please try as a first step, to go to your App settings, -> Profile Settings -> Clear local data and after log in again?
Please let us know if it changes anything.

Thank ypu

**Comment by Germanbight** [comment]
Problem still exits, I have cleared local data too and logged in again.the "about and install" option on the menu bar, crashes the app.I can't see the preview of 2.16.0

--
Securely sent with Tutanota. Claim your encrypted mailbox today!
https://tutanota.com <https://tutanota.com>

11. Jul 2018 07:30 by notifications@github.com <mailto:notifications@github.com>:

**Comment by Willow-Rose** [comment]
Problem still exists for me as well even though I've cleared all data and logged in again. Where do I find 2.16.0?

**Comment by ThatElfGuy** [comment]
@Willow-Rose You can download 2.16 here: https://github.com/fetlife/android/releases/download/v2.16.0/fetlife-2.16.0.apk

**Comment by ThatElfGuy** [comment]
@Willow-Rose @Germanbight can you let me know what android version you're using?

**Comment by Willow-Rose** [comment]
I have a Samsung galaxy s3. Is that what you're asking?

**Comment by ThatElfGuy** [comment]
@Willow-Rose you should be able to find the Android version in Settings - About Phone/Device

**Comment by Willow-Rose** [comment]
4.4.2

**Comment by Willow-Rose** [comment]
No images on 2.16.0 for me

**Comment by Germanbight** [comment]
I've  Android 4.3 
loaded 2.16  no change, still no opics 
Ta

**Comment by scruffynerf** [comment]
Android v4.4.2 fresh reinstall of 2.15

Helpful debug: images won't display AND attempting to find app update fails.  Likely something is broken in the net code, or maybe Fet security changes broke something?

**Comment by ThatElfGuy** [comment]
I have a hypothesis that I'm working on with @kistitan 

The problem seems to be related to android versions 4.4.x and older - and their relationship with TLS 1.2

Apologies for the no-images issue, but hopefully we can get to the bottom of it and find a resolution soon :)

**Comment by Germanbight** [comment]
Thank you, look forward to resolution!
--
Securely sent with Tutanota. Claim your encrypted mailbox today!
https://tutanota.com <https://tutanota.com>

13. Jul 2018 14:28 by notifications@github.com <mailto:notifications@github.com>:

**Comment by ThatElfGuy** [comment]
@Willow-Rose @Germanbight @scruffynerf 

Can you download the new preview version and let us know if the issue is resolved?

**Comment by Willow-Rose** [comment]
Is the preview version 2.16? If so, I've already downloaded it and still have no images.

**Comment by ThatElfGuy** [comment]
@kistitan Back to the drawing board!

**Comment by ThatElfGuy** [comment]
https://github.com/fetlife/android/releases/download/v2.16.4/fetlife-2.16.4.apk

**Comment by Willow-Rose** [comment]
Still no images and getting the message "Server could not be reached. Please check your internet connection." again even though my internet connection is completely fine.

**Comment by FernandoMiguel** [comment]
I can confirm that 16.4 is showing the warning about can't reach server far more often than previous versions

OTOH images are showing for me, although some are taking much longer than before

**Comment by kistitan** [comment]
@Willow-Rose 
That's bad... than we are a bit out of luck.
We used a solution that is claimed to enable by default disabled Tls version on phone with older OS, but appearently it does not work for you...

We will give another try to find a workaround, but I am afraid sooner than later we will just have to supporting OS versions prior to Lollipop (5.0)

**Comment by scruffynerf** [comment]
No, 2.16.4 doesn't solve the problem.  Visiting fetlife in a browser does work, so its not that Kitkat can't be made to work, but clearly enabling TLS 1.2 takes more effort.

**Comment by ThatElfGuy** [comment]
Hey @scruffynerf @Willow-Rose @Germanbight 

Can you try the new version? https://github.com/fetlife/android/releases/download/v2.16.7/fetlife-2.16.7.apk

**Comment by scruffynerf** [comment]
it works!  nice job @kistitan

**Comment by Germanbight** [comment]
Sadly i get a parse error and can't upload the file 

--
Securely sent with Tutanota. Claim your encrypted mailbox today!
https://tutanota.com <https://tutanota.com>

20. Jul 2018 18:09 by notifications@github.com <mailto:notifications@github.com>:

**Comment by scruffynerf** [comment]
@Germanbight do you mean download the file (New apk)? I found trouble using downloader apps, but chrome and Brave worked for that url.  weirdness with github I think.

**Comment by Germanbight** [comment]
Sadly I'm having some difficulty!I've tried with Brave, Chrome, Firefox and Dolphin and all come up with parsing error :(


--
Securely sent with Tutanota. Claim your encrypted mailbox today!
https://tutanota.com <https://tutanota.com>

22. Jul 2018 01:40 by notifications@github.com <mailto:notifications@github.com>:
